app/views/appointments/calendar.html.erb

```diff
@@ -22,8 +22,8 @@
             </p>
           </div>
         </div>
-      <% end %>
+     <% end %>
```

Comment:
Indentation.

Reply:
hmm :/ I'm not sure, it seems correct to me but maybe I'm not seeing something